src/app/shared/helpers/input-is-not-null-or-undefined.ts

```diff
@@ -0,0 +1,3 @@
+export function inputIsNotNullOrUndefined<T>(input: null | undefined | T): input is T {
```

Comment:
👍  let's call it `isNotNullNorUndefined` (update file name as well)

Reply:
Fixed